index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio</title>
+        <script src="https://kit.fontawesome.com/8a30dfc1c1.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="style.css">
+
+    </head>
+
+      <div class="headline-wrapper">
+        <div class="app-wrap">
+        
+          <header class="app-header">
+            <a href="#" class="button">
+              <i class="fas fa-bars"></i>
+            </a>
+            
+            <h1>My Logo</h1>  
+        
+          </header>
+
+         <div class="content">
+           <h2>I'm Vana<br> Glad to see<br> you!</h2>
+           <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          
+            <h3>LET'S CONNECT</h3>
+         </div>
+        </div>
+
+        <div class="icon-bar">
+          &ensp;<a href="#">
+            <img title="Twitter" alt="Twitter" src="file:///C:/Users/VS47~1/AppData/Local/Temp/Vector.svg"/>         
+          </a>&ensp;
```

Comment:
- [ ] Please kindly fix the social pages link images, currently they are broken:

![image](https://user-images.githubusercontent.com/77038610/143388329-6b8a9b54-6701-4a70-a9f8-d84653840772.png)

These links are broken because they are located on your personal desktop not the repository, I kindly suggest you to add them to your repo and then fix the path 🤗

- [ ] Links should be inside `<ul> <li>` tags:
```
<ul>
  <li> <a href="#"><img src="img/icon/twitter.svg" alt="Twitter Icon"></a>  </li>
  <li> <a href="#"><img src="img/icon/linkedin.svg" alt="Linkedin Icon"></a>  </li>
  ...
</ul>
```
Please kindly add them inside proper tags 👍🏻 

- [ ] Links are detached from the body: 

![image](https://user-images.githubusercontent.com/77038610/143388820-ae38c196-8c73-4604-89c6-ee8f1745ae42.png)
Please kindly add them below `LET'S CONNECT`